modules/Bio/EnsEMBL/Compara/PipeConfig/UpdateReferenceDatabase_conf.pm

```diff
@@ -359,6 +372,28 @@ sub core_pipeline_analyses {
             },
         },
 
+        {   -logic_name => 'orthofinder_factory',
+            -module     => 'Bio::EnsEMBL::Compara::RunnableDB::PassFastaDumpsPerCollection',
+            -parameters => {
+                'symlink_dir'          => $self->o('shared_fasta_dir'),
+                'ref_member_dumps_dir' => $self->o('ref_member_dumps_dir'),
+                'compara_db'           => $self->o('ref_db'),
+            },
+            -flow_into  => {
+                2 => [ 'run_orthofinder' ],
+            },
+        },
+
+        {   -logic_name => 'run_orthofinder',
+            -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+            -parameters => {
+                'orthofinder_exe' => $self->o('orthofinder_exe'),
+                'orthofinder_dir' => $self->o('orthofinder_dir'),
+                'cmd'             => 'cd #orthofinder_dir#; #orthofinder_exe# -f #symlink_dir#',
```

Comment:
Based on my understanding and how I installed it, I think cd-ing into OrthoFinder's folder is no longer required. Furthermore, OrthoFinder's hardware detection doesn't work well in the farm (it picks the node's specs, not the assigned ones), so the number of cores has to be specified:
```suggestion
                'cmd'             => '#orthofinder_exe# -t 16 -a 8 -f #symlink_dir#',
```
(_Note:_ the value for `-a` is based on the suggestion of the documentation to use 4-8x smaller values, but David Emms used 8 for both in one of his examples in the emails)

Reply:
Okay, good to know, thanks - this is why I kept asking you if the usage was exactly the same!